### ISSUE TITLE
same logic twice

### ISSUE THREAD

**Issue by klaus7** [jmix-framework/jmix#2113]
https://github.com/jmix-framework/jmix/blob/e6abeaa27cc4af4ffb8d22a9a2cc736520b75198/jmix-flowui/flowui/src/main/java/io/jmix/flowui/model/impl/DataContextImpl.java#L703

This is already done in `updateFetchPlans(saveContext);`

**Comment by klaus7** [comment]
There is actually another problem here that leads to a significant performance issue. If we don't care about the reloading of the saved entities and have a large count of entities in the save context, the updateFetchPlans method requires a lot of CPU time.

**Comment by knstvk** [comment]
> If we don't care about the reloading of the saved entities and have a large count of entities in the save context, the updateFetchPlans method requires a lot of CPU time.

Agree. But this is a different problem which is not so easy to fix. Could you create a separate issue?

**Comment by klaus7** [comment]
> > If we don't care about the reloading of the saved entities and have a large count of entities in the save context, the updateFetchPlans method requires a lot of CPU time.
> 
> Agree. But this is a different problem which is not so easy to fix. Could you create a separate issue?

Sure, will do that.
